### PR TITLE
config-generator: CNF Installation (2.2) Add generator for v2 config

### PR DESCRIPTION
**Pull request by kosstennbl**
## Description
Create new config generation task, that would generate v2 config.
This task should be more interactable and friendly to new testsuite users.

## Issues:
Refs: https://github.com/cnti-testcatalog/testsuite/issues/2138

Re-created from different branch, old PR: #2142

### CODE REVIEW

File: src/tasks/cnf_setup.cr [hunk 1]
```diff
@@ -99,3 +104,67 @@ end
 task "sample_generic_cnf_cleanup" do |_, args|
   CNFManager.sample_cleanup(config_file: "sample-cnfs/sample-generic-cnf", verbose: true)
 end
+
+def interactively_create_config
+  config = {
+    config_version: "v2",
```
Comment (martin-mat) [inline src/tasks/cnf_setup.cr:110]: I'd suggest not to hardocde "v2" here, rather use the latest version config we have in one of prior changes,

File: src/tasks/cnf_setup.cr [hunk 2]
```diff
@@ -99,3 +104,67 @@ end
...
+    else
+      puts "Invalid choice. Please try again."
+    end
+  end
+
+  yaml_config = config.to_yaml
+  puts "Generated Configuration:"
+  puts yaml_config
+
+  output_file = prompt("Choose output config path (leave empty for ./cnf-testsuite.yml): ")
```
Comment (martin-mat) [inline src/tasks/cnf_setup.cr:159]: use CONFIG_FILE instead of "cnf-testsuite.yml"

File: src/tasks/cnf_setup.cr [hunk 3]
```diff
@@ -99,3 +104,67 @@ end
...
+  if output_file.strip.empty?
+    output_file = "cnf-testsuite.yml"
```
Comment (martin-mat) [inline src/tasks/cnf_setup.cr:161]: use CONFIG_FILE instead of "cnf-testsuite.yml"

**Review by martin-mat — CHANGES_REQUESTED** [review]

**Review by svteb — APPROVED** [review]
lgtm

**Review by martin-mat — APPROVED** [review]
lgtm

**Review by Smitholi67 — APPROVED** [review]
LGTM

**Pull request merged**